docs/rules/sort-attributes.md

```diff
@@ -26,6 +26,7 @@ Default:
 
 ```astro
 ---
+/* eslint astro/sort-attributes: "error" */
```

Comment:
Without this line there are no errors. I used https://ota-meshi.github.io/eslint-plugin-astro/rules/prefer-split-class-list/ as a reference of a correct version